doc/frameworks/pytorch/using_pytorch.rst

```diff
@@ -547,6 +548,26 @@ block, for example:
         with torch.jit.optimized_execution(True, {"target_device": "eia:0"}):
             output = model(input_data)
 
+If you use Elastic Inference with torch version >= 1.5.1, please implement your own ``predict_fn`` like below.
+
+.. code:: python
+
+    import torch
+    import numpy as np
```

Comment:
nit: apply [`isort`](https://pypi.org/project/isort/) to imports

Reply:
fix in rev2